src/components/Cover/index.js

```diff
@@ -1,5 +1,11 @@
+import { StyledCover } from "./cover.styles";
+
 const Cover = (props) => {
-  return <img src={props.coverUrl} alt={props.title} />;
```

Comment:
destructuring... The code will look so much cleaner if you apply destructuring everywhere you can, I'd definitely try and do it!